pkgs/objective_c/README.md

```diff
@@ -0,0 +1,4 @@
+# objective_c
+
+A library to access Objective C from Dart and Flutter that acts as a support
+library for package:ffigen.
```

Comment:
Do we need an issue template for this? If no, please update the existing issue template for package FFIgen to also point tot his new package.

Reply:
Probably best to give it its own issue template